source/MaterialXGenShader/ShaderTranslator.h

```diff
@@ -0,0 +1,76 @@
+//
+// TM & (c) 2017 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
```

Comment:
For newly created files like this one, let's go ahead and use "2020" as the copyright year.  Going forward, I think the best approach will be to mark each file with the year in which it was first added to the codebase, rather than requiring uniformity across files.